gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.1.1-bin.zip
```

Comment:
This should be left out of the PR, we have an issue open at paperweight (https://github.com/PaperMC/paperweight/issues/55) as Gradle 7.1.1 added some new deprecations. Until that issue is resolved, I'd prefer to not spam everyone's builds with warnings.

Reply:
I went back to Gradle 7.1